website/content/en/preview/AWS/provisioning.md

```diff
@@ -150,6 +150,21 @@ spec:
       httpTokens: required
 ```
 
+### Amazon Machine Image (AMI) Family
+
+The AMI used when provisioning nodes can be controlled by the `amiFamily` field. Based on the value set for `amiFamily`, Karpenter will automatically query for the appropriate [EKS optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) via AWS Systems Manager (SSM). 

```

Comment:
We should be clear about the AMI that we'll be resolving here. For AL2 it's the EKS Optimized AMI, but this reads as though there's Bottlerocket and Ubuntu variants of the optimized AMI as well, which isn't the case.